### PR TITLE
Lots more GPU / CPU K8S cluster updates

### PR DESCRIPTION
**Pull request by Kashomon**
- Make parallelism a var since we can calculate it.
- Make the default GPU script the small one to match the CPU one
- Move the docs to cluster/README.md
- Use beta GPUs instead of alpha GPUs since that's what API we're using.

### CODE REVIEW

File: cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+# Kubernetes Cluster and Image Management
+
+As you might notice, playing games is fairly slow.  One way to speed up playing
```
Comment (amj) [inline cluster/README.md]: Now without a segue from the previous paragraph ;)  Maybe rephrase

File: cluster/cluster-up-cpu [hunk 2]
```diff
@@ -9,12 +9,17 @@ echo "Using Project:      ${PROJECT}"
 echo "Using Zone:         ${ZONE}"
 echo "Using Cluster Name: ${CLUSTER_NAME}"
 echo "Using K8S Version:  ${K8S_VERSION}"
+echo "Number of Nodes:    ${NUM_K8S_NODES}"
+
+export PARALLELISM="$((3 * ${NUM_K8S_NODES}))"
```
Comment (amj) [inline cluster/cluster-up-cpu]: Eep.  Having parallelism defined as an env var is great when N is integers greater than one -- how's your bash math with floating points?  (For GPU, parallelism is usually 98% or 95% of number of nodes)

File: cluster/cluster-up-cpu [hunk 3]
```diff
@@ -9,12 +9,17 @@ echo "Using Project:      ${PROJECT}"
...
+
+check_gcloud_exists
 
 # Create a Kubernetes cluster
 # Note, we require Intel Broadwells since they are a bit newer, and can provide
 # up to a 30% speedup, since we're so CPU bound.
 gcloud beta container clusters create \
-  --num-nodes 3 \
+  --num-nodes $NUM_K8S_NODES \
   --machine-type n1-standard-8 \
```
Comment (amj) [inline cluster/cluster-up-cpu]: if these are standard-8's, parallelism should probably be higher, in 6-8

File: README.md [hunk 4]
```diff
@@ -319,260 +319,7 @@ Additionally, you can follow along with the training progress with TensorBoard -
...
-  ```
-
-* Monitoring how long it's taking the daemonset to install the nvidia driver on
-  your nodes
-  ```
-  kubectl get no -w -o yaml | grep -E 'hostname:|nvidia-gpu'
-  ```
-
-
-If you've run rsync to collect a set of SGF files (cheatsheet: `gsutil -m cp -r
```
[the remark this answers is not in the document]
Comment (Kashomon) [reply README.md]: Sounds totally reasonable. This doc move was pretty coarse.

File: cluster/README.md (hunk 1 above)
Comment (Kashomon) [reply cluster/README.md]: Removed segue

File: cluster/cluster-up-cpu (hunk 2 above)
Comment (Kashomon) [reply cluster/cluster-up-cpu]: I think Parallelism is an int in the Kubernetes job api

File: cluster/cluster-up-cpu (hunk 3 above)
Comment (Kashomon) [reply cluster/cluster-up-cpu]: Changed to 8

Comment (Kashomon) [reply cluster/cluster-up-cpu]: (I'll do some test also)
